ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -154,6 +154,64 @@ presubmits:
             path: kubeconfig
           secretName: build-farm-credentials
     trigger: (?m)^/test( | .* )build01-dry,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: app.ci
+    context: ci/build-farm/build02-dry
+    decorate: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-release-master-build02-dry
+    rerun_command: /test build02-dry
+    spec:
+      containers:
+      - args:
+        - --config-dir
+        - clusters/build-clusters/02_cluster
+        - --kubeconfig
+        - /etc/build-farm-credentials/kubeconfig
+        - --context
+        - build02
```

Comment:
The context options is not needed, there is only one kubeconfig (It might even break the thing, I think the context name is `config-updater` or some such)

Reply:
This is copied from the job for `build01` and i checked

```
oc --kubeconfig ~/Downloads/tmp.NQcmihO2/sa.config-updater.build02.config config get-contexts
CURRENT   NAME      CLUSTER                                 AUTHINFO   NAMESPACE
*         build02   api-build02-gcp-ci-openshift-org:6443   build02    ci
```

Reply:
Removed.